nixos/modules/services/printing/cupsd.nix

```diff
@@ -384,14 +384,14 @@ in
         preStart = lib.optionalString cfg.stateless ''
           rm -rf /var/cache/cups /var/lib/cups /var/spool/cups
         '' + ''
-            mkdir -m 0700 -p /var/cache/cups
-            mkdir -m 0700 -p /var/spool/cups
-            mkdir -m 0755 -p ${cfg.tempDir}
+            mkdir -m 0700 /var/cache/cups || true
+            mkdir -m 0700 /var/spool/cups || true
+            mkdir -m 0755 ${cfg.tempDir} || true
```

Comment:
This might fail for certain paths

Reply:
true, maybe we should do an exception here